src/todo/todo.lua

```diff
@@ -65,9 +67,84 @@ function todo.save_task(task, should_add_to_top)
 
     table.insert(global.todo.open, add_index, task)
 
+    todo.update_export_dialog_button_state()
+
     return task
 end
 
+function todo.update_export_dialog_button_state()
+    local task_count = #global.todo.open + #global.todo.done
+
+    for _, player in pairs(game.players) do
+        local main_frame = todo.get_main_frame(player)
```

Comment:
```suggestion
        local player_main_frame = todo.get_main_frame(player)
```
This variable name should reflect the fact that this is the main frame a of a player, not a local main_frame.